internal/commands/server/list.go

```diff
@@ -46,14 +66,91 @@ func (s *listCommand) ExecuteWithoutArguments(exec commands.Executor) (output.Ou
 		})
 	}
 
+	columns := []output.TableColumn{
+		{Key: "uuid", Header: "UUID", Colour: ui.DefaultUUUIDColours},
+		{Key: "hostname", Header: "Hostname"},
+		{Key: "plan", Header: "Plan"},
+		{Key: "zone", Header: "Zone"},
+		{Key: "state", Header: "State"},
+	}
+
+	if ls.showIPAddresses.Value() {
+		ipaddressMap, err := getIPAddressesByServerUUID(svc)
+		if err != nil {
+			return nil, err
+		}
+
+		for i, row := range rows {
+			uuid := row[0].(string)
+
+			var listIpaddresses []listIPAddress
+			if apiIpaddresses, ok := ipaddressMap[uuid]; ok {
+				for _, ipa := range apiIpaddresses {
+					listIpaddresses = append(listIpaddresses, listIPAddress{
+						Access:   ipa.Access,
+						Address:  ipa.Address,
+						Floating: ipa.Floating.Bool(),
```

Comment:
Current IP address list that GUI gets from `https://gate.upcloud.com/v2/server` does not include `floating` field.